docs/edge-compute/edge-deployments.md

```diff
@@ -14,19 +14,19 @@ Edge deployments can be managed on a per-device basis or a per-workflow basis.
 
 After selecting a device from your device list, click "Edge Compute" in the left navigation. This will bring you to a page where you can view the Edge Agent version currently deployed to the device, as well as a list of all workflows deployed to the device and their version status.
 
-Clicking the "Deploy" button here will allow you to select a workflow, and then select an existing version of that workflow to deploy to the device you're editing. If you choose a workflow that is already deployed to the device (or is scheduled to be deployed), you will have the option of changing the version of the workflow to use as if you had clicked the "Settings" icon for that workflow in the list below the button.
+Clicking the "Settings" (gear) icon in any deployment table row will allow you to edit that specific deployment, including changing the workflow version or canceling a scheduled rollout.
 
-![Device Deployment Modal](/images/edge-compute/device-deployment-modal.png "Device Deployment Modal")
+Clicking the "Deploy" button here will allow you to select a workflow, and then select an existing version of that workflow to deploy to this device. If you choose a workflow that is already deployed (or is scheduled to be deployed) to the device, you will have the option of changing the version of the workflow to use as if you had clicked the "Settings" icon for that workflow in the list below the button.
 
-Clicking the "Settings" (gear) icon in any deployment table row will allow you to edit that specific deployment, including changing the workflow version or canceling a scheduled rollout.
+![Device Deployment Modal](/images/edge-compute/device-deployment-modal.png "Device Deployment Modal")
 
 ### Managing Per Workflow
```

Comment:
I think it would be nice if this was a ## so it would be linkable and visible in the side bar. Just a nit.